src/lib/server/test.test.ts

```
@@ -0,0 +1,194 @@
+				headers: { accept: 'application/json' }
+			}
+		);
+	});
+});
+
+describe('Fetch test timer', () => {
+	const candidate_test_id = 24;
+	const candidate_uuid = 'candidate_uuid';
+
+	it('should throw error if candidate test id and uuid are not passed', async () => {
+		await expect(getTimeLeft(null as any, '')).rejects.toThrow(/are required/i);
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)